runtime/ms-rest/index.d.ts

```diff
@@ -16,23 +16,40 @@ export interface ClientRequestOptions extends RequestOptions {
 }
 
 /**
+ * HttpResponse wrapper that encapsulates the
+ * response stream plus available status codes/messages
+ *
+ * @property {Object} headers       - response headers returned from the server, duplicate filtered
+ *                                    as described in https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_message_headers
+ * @property {Array} [rawHeaders]   - Original unprocessed response headers
+ * @property {Number} statusCode    - HTTP status code for response
+ * @property {String} statusMessage - HTTP status response message
+ */
+export interface Response extends stream.Readable {
```

Comment:
Rather than defining our own type, it would be better to expose the RequestResponse interface from the request typings.
We have a dependency on `@types/request`.

```typescript
import { RequestResponse } from 'request';

. . . 
export interface HttpOperationResponse<T> {
  request: WebResource;
  response: RequestResponse;
  body: T;
}
 . . .
```